### PR TITLE
Updating dependencies and php version

### PR DESCRIPTION
**Pull request by weaverryan**
Currently blocked by https://github.com/Nyholm/git-reviewer/issues/4 as we have one GH action that runs both Carson AND that library at the same time. And so, they both need to support 8.1

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -40,7 +40,7 @@
     },
     "config": {
         "platform": {
-            "php": "7.4"
+            "php": "8.1."
```
Comment (OskarStark) [inline composer.json:43]:
```suggestion
            "php": "8.1"
```
?

**Comment by Nyholm** [comment]
Let's drop the git-reviewer feature.

**Comment by Nyholm** [comment]
PR is ready for a review.

**Review by OskarStark — APPROVED** [review]
Should we switch to 8.2 ?

**Comment by Nyholm** [comment]
Yes. But I let that be the next PR

**Pull request merged**